### PR TITLE
XCORNER-7 reviews widget

### PR DESCRIPTION
**Pull request by mgbelegu**
#### Changes:

1. Created the reviews widget

### CODE REVIEW

File: widget-templates/Reviews/widget.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        }
+    }
+</style>
+    
+<section class="js-review-swiper c-carousel c-reviews-carousel c-reviews-carousel__{{_.id}}">
+    <div class="swiper-wrapper">
+        {{#each slides}}
+            <div class="swiper-slide">
+                <div class="c-review-carousel__card" data-rating="{{rating}}">
+                    <div class="c-review-carousel__rating-stars"></div>
```
Comment (marcinkrzeminski) [inline widget-templates/Reviews/widget.html]: @mgbelegu please add aria attributes to make this more accessible

File: assets/scss/widgets/reviews.scss [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            max-width: 100%;
+        }
+    }
+}
+
+.c-review-carousel__card {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    gap: 24px;
```
Comment (marcinkrzeminski) [inline assets/scss/widgets/reviews.scss]: @mgbelegu please replace all px values to rems using `rem-calc()`

**Review by marcinkrzeminski — CHANGES_REQUESTED** [review]
@mgbelegu some minor adjustments, mostly px to rem

**Comment by mgbelegu** [comment]
@marcinkrzeminski Hi, can you please review again?

**Review by marcinkrzeminski — APPROVED** [review]

**Pull request merged**